engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/definition/validator/ValidatorsExtractor.scala

```diff
@@ -0,0 +1,18 @@
+package pl.touk.nussknacker.engine.definition.validator
+
+import java.lang.reflect.Parameter
+
+import pl.touk.nussknacker.engine.api.definition.ParameterValidator
+
+object ValidatorsExtractor {
+  def extract(parameter: Parameter): List[ParameterValidator] = {
+    new ValidatorsExtractor(List(MandatoryValueValidatorExtractor)).extract(parameter)
+  }
+}
+
+class ValidatorsExtractor(validatorExtractors: Iterable[ValidatorExtractor]) {
```

Comment:
is this class really needed?

Reply:
Inlined in `object ValidatorsExtractor`

Reply:
I think that @mproch want to remove this class at all and inline in place of usage, but for me can be like now also